distributed/http/worker/prometheus/core.py

```diff
@@ -161,6 +161,38 @@ def collect(self):
             value=self.server._tick_counter,
         )
 
+        # This duplicates spill_time_total; however the breakdown is different
+        evloop_blocked_total = CounterMetricFamily(
+            self.build_name("event_loop_blocked_time_total"),
+            "Total time during which the worker's event loop was blocked "
+            "by spill/unspill activity since the latest worker reset",
+            labels=["cause"],
+        )
+        # This is typically higher than spill_time_per_key_max, as multiple keys can be
+        # spilled/unspilled without yielding the event loop
+        evloop_blocked_max = GaugeMetricFamily(
+            self.build_name("event_loop_blocked_time_max"),
+            "Maximum contiguous time during which the worker's event loop was blocked "
+            "by spill/unspill activity since the previous Prometheus poll",
+            labels=["cause"],
+        )
```

Comment:
I left the name of the measure generic so that we can add a tap into `GCDiagnosis` in a later PR, to measure garbage collection.